native/cocos/renderer/core/PassUtils.h

```diff
@@ -113,4 +113,7 @@ bool overrideMacros(MacroRecord &target, const MacroRecord &source);
 
 MaterialProperty toMaterialProperty(gfx::Type type, const ccstd::vector<float> &vec);
 
+bool macroRecordAsBool(const MacroRecord::mapped_type &v);
+ccstd::string macroRecordAsString(const MacroRecord::mapped_type &v);
```

Comment:
How about rename to `macroRecordToBool` and `macroRecordToString `?

Reply:
I think `As` means that it's a bool in general, but if it's not , convert it.
`To` means the source is not a `bool`, let's convert it.
@PatriceJiang What do you think? The function name is named by you originally.

Reply:
Why i suggest to use `to` is because it convert from `MacroRecord::mapped_type` to another type.